src/components/NotFound.tsx

```diff
@@ -0,0 +1,29 @@
+import React, {FunctionComponent, useState} from "react";
+import {RouteComponentProps} from "@reach/router";
+import ReportModal from "./modal/components/report/ReportModal";
```

Comment:
Is grouping imports rule isn't enabled yet?

Reply:
I'm not sure. Can you check if there are added to hipo/base or hipo/react eslint rules ? If not, we can create a new issue to add them there

Reply:
Yeah, filed it https://github.com/Hipo/eslint-config-hipo-base/issues/8